src/activity-functions-function-notation.xml

```diff
@@ -775,7 +889,107 @@
+                      <static>1</static>
+                  </var>
+                  <var name="$solutions2">
+                      <static>2.64</static>
+                  </var>
+                  <var name="$maxNumSolutions">
+                      <static>3</static>
+                  </var>
+                   <var name="$minNumSolutions">
+                      <static>1</static>
+                  </var>
+
+
+                  <pg-code>
+                      $numZeros = 3;
+                      $zeros = List(-4,-1,2);
+                      $numSolutions1 = 2;
+                      $solutions = List(-2.7,2.46);
+                      $numSolutions2 = 1;
+                      $solutions2 = List(2.64);
+                      $maxNumSolutions = 3;
+                      $minNumSolutions = 1;
+
+                  </pg-code>
+              </setup>
+
+              <stage>
+                  <statement>
+                      <p>On a graph the output is the height of the graph.  Therefore,<q>Solve <m>f(x) = 0</m></q> on a graph means, <q>Find all the places on the <m>x</m> axis where the graph reaches a height of <m>y = 0</m></q></p>
+                      <p>Drag the output to the desired height on the y-axis to help you solve the given equations.</p>
+
+                      <p>[@MODES(HTML=&gt;'&lt;iframe scrolling="no" src="https://www.geogebra.org/material/iframe/id/DpUNTRSC/width/413/height/331/border/888888" width="413px" height="331px" style="border:0px;"&gt; &lt;/iframe&gt;', TeX =&gt;"A GeoGebra applet.");@]*</p>
+
+                      <p>To solve the equation <m>f(x) = 0</m> graphically, move the height of the horizontal line to <m>y = 0</m></p>
+
+                      <p>The horizontal line intersects with <m>f(x)</m> <var name="$numZeros" width="5" /> times.</p>
+
+                      <p>The solutions to <m>f(x) = 0</m> are <m>x=</m><var name="$zeros" width="15" />. (Use commas.)</p>
+
+                      <p>Notice that all the y values are 0, but the x values are diffenent.  The x values are the <em>solutions to the equation <m>f(x) = 0</m></em>.</p>
+                  </statement>
+              </stage>
+
+              <stage>
+                <statement>
+                  <p>[@MODES(HTML=&gt;'&lt;iframe scrolling="no" src="https://www.geogebra.org/material/iframe/id/DpUNTRSC/width/413/height/331/border/888888" width="413px" height="331px" style="border:0px;"&gt; &lt;/iframe&gt;', TeX =&gt;"A GeoGebra applet.");@]*</p>
+
+                  <p>Move the horizontal line to <m>y = 2</m>. Use this line to solve the equation <m>f(x) = 2</m></p>
+
+                  <p>How many solutions does the equation have? If there are none, type <q>none</q>.</p>
+                  <p>The equation has <var name="$numSolutions1" width="5" /> solution(s).</p>
+
+                  <p>Use the x coordinates of the points on the horizontal axis to determine all the solutions to the equation.</p>
```

Comment:
All of your comments have been addressed except for the problem I am having with the ordered list I mentioned above.  I'm off to work on the remaining activities for this chapter.